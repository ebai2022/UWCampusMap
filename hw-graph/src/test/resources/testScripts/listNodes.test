# Test node listing

# Test empty case
CreateGraph A
ListNodes A

# Test single case
AddNode A e1
ListNodes A

# Test multi case
AddNode A e2
AddNode A e3
ListNodes A