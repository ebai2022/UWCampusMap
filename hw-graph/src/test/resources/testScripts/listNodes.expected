# Test node listing

# Test empty case
created graph A
A contains:

# Test single case
added node e1 to A
A contains: e1

# Test multi case
added node e2 to A
added node e3 to A
A contains: e1 e2 e3